standard-app/templates/workloads/cronjob.yaml

```diff
@@ -96,7 +96,8 @@ spec:
               image: "{{ $.Values.image.repository }}/{{ if $cronjobConfig.imageName }}{{ $cronjobConfig.imageName }}
                                                       {{- else if $.Values.image.name }}{{ $.Values.image.name }}
                                                       {{- else }}{{ $container.name }}{{ end }}:
-                                                      {{- if  $cronjobConfig.imageTag }}{{ $cronjobConfig.imageTag }}
+                                                      {{- if $container.tag }}{{ $container.tag }}
+                                                      {{- else if $cronjobConfig.imageTag }}{{ $cronjobConfig.imageTag }}
```

Comment:
let's be consistent with the names and use `tag` instead of 'imageTag' and 'image' instead of image name everywhere.